pkg/KubeArmorMacro/config/crd/bases/security.accuknox.com_kubearmormacroes.yaml

```diff
@@ -0,0 +1,68 @@
+
+---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.3.0
+  creationTimestamp: null
+  name: kubearmormacroes.security.accuknox.com
```

Comment:
typo: macroes => macros

Reply:
I don't known why but kubebuilder still selecting the plural version for this field.
macros -> macros; policy -> policies, etc.

should we change anyway?